website/docs/search.md

```diff
@@ -27,7 +27,7 @@ Docusaurus has **official support** for [Algolia DocSearch](https://docsearch.al
 
 The service is **free** in most cases: just [apply to the DocSearch program](https://docsearch.algolia.com/apply).
 
-It works by crawling the content of your website every 24 hours and putting all the content in an Algolia index. This content is then queried directly from your front-end using the Algolia API.
+It works by regularly crawling the content of your website (the schedule is configurable from the web interface) and putting all the content in an Algolia index. This content is then queried directly from your front-end using the Algolia API.
```

Comment:
```suggestion
It crawls your website once a week (the schedule is configurable from the web interface) and index all the content in an Algolia index. This content is then queried directly from your front-end using the Algolia API.
```

A bit repetitive with the `index` word but I feel like this is more appropriate, since they will see those terms quite often while browsing our interfaces